integration-tests/README.md

```diff
@@ -184,7 +98,7 @@ externalRestEnabled: true
 javaLoggingLevel: FINE
 ```
 
-src/integration-tests/resources/domainonpvwlst.yaml - input/customized properties for PV/Load Balancer/WebLogic Domain. Any property can be provided here from kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-domain-inputs.yaml and kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/create-pv-pvc-inputs.yaml. For all the properties that are not defined here, the default values in the sample inputs are used while generating inputs yaml.
+src/integration-tests/resources/domainonpvwlst.yaml - See "Input Yaml to the test". For all the properties that are not defined here, the default values in the sample inputs are used while generating inputs yaml.
```

Comment:
you can make a link if you want by putting `[Input Yaml to the test](#input-yaml-to-the-test)`

Reply:
Done